frontend/src/app.js

```diff
@@ -20,6 +22,8 @@ function App() {
           {isSignedIn && <Route path='/user' element={<User />} />}{' '}
           <Route path='/books' element={<ViewAllBooks />} />
           <Route path='/books/:book_id' element={<ViewBook />} />
+          {isSignedIn && <Route path='/collection/:user_id' element={<ViewCollection />} />}{' '}
```

Comment:
in the future, the userid should be removed from the url as if we leave this, it means a user could manipulate another user's collection knowing their user_id

the book-id should be removed to if possible
also, i think it's better to have the add-book url be /addbook once u've removed the params

Reply:
oki noted. will resolve the user id issue in another PR